content/en/examples/examples_test.go

```diff
@@ -600,6 +600,12 @@ func TestExampleObjectSchemas(t *testing.T) {
 			"node-problem-detector-configmap": {&apps.DaemonSet{}},
 			"termination":                     {&api.Pod{}},
 		},
+		"dra": {
+			"deviceclass":	{&resource.DeviceClass{}},
+      "resourceclaim": {$resource.ResourceClaim{}},
```

Comment:
```suggestion
      "resourceclaim": {&resource.ResourceClaim{}},
```

Does this test run somewhere? This should have failed to compile.

There's probably also a missing import of the package.

Reply:
It's also broken on main:
```console
$ git checkout origin/main 
Previous HEAD position was 8f0424d057 blog: local storage features go beta (#27240)
HEAD is now at 8f48a396c1 Merge pull request #51526 from my-git9/npa-32177
pohly@pohly-desktop:/nvme/gopath/src/k8s.io/website$ go test ./content/en/examples/
go: updates to go.mod needed; to update it:
	go mod tidy
$ go mod tidy
go: finding module for package k8s.io/apimachinery/pkg/api/operation
go: finding module for package k8s.io/apimachinery/pkg/api/safe
go: finding module for package k8s.io/apimachinery/pkg/api/validate
go: finding module for package k8s.io/api/resource/v1beta2
go: k8s.io/website/content/bn/examples tested by
	k8s.io/website/content/bn/examples.test imports
	k8s.io/kubernetes/pkg/apis/apps/install imports
	k8s.io/kubernetes/pkg/apis/apps/v1beta1 imports
	k8s.io/apimachinery/pkg/api/operation: module k8s.io/apimachinery@latest found (v0.33.2, replaced by k8s.io/apimachinery@v0.32.0), but does not contain package k8s.io/apimachinery/pkg/api/operation
...
```

Let's not make it worse, but fixing this is outside of the scope for this PR.